curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -17,439 +17,479 @@ To pass the quiz, you must correctly answer at least 17 of the 20 of the questio
 
 #### --text--
 
-Placeholder question
+Which html element allows a two-dimensional table with column and row?
 
 #### --distractors--
 
-Placeholder distractor 1
+`row`
 
 ---
 
-Placeholder distractor 2
+`matrix`
 
 ---
 
-Placeholder distractor 3
+`table` using attributes `r="3" c="4"`
 
 #### --answer--
 
-Placeholder answer
+`table`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What elements are used to specify row and header?
```

Comment:
```suggestion
What elements are used to specify a row and a row header?
```

couldn't `thead` be considered a header? I thought this makes it a little more specific.